wordpress-mobile/WordPress-Android
# Stories: Remove gallery fallback

Before this PR we defined an allowlist of alpha testing sites, which supported the Jetpack story block. All other sites would have their stories uploaded as a gallery.

This PR eliminates the allowlist and the fallback - all sites we deem to support stories will have stories uploaded as a Story block.

**This PR is a follow-up to #12728 and should be merged after that one.**

### To test:
For various test sites (that meet the requirements outlined in #12728), try uploading a story. Verify that the story is created and uploaded successfully as a Story block.

In order to actually view the story on the web, there are a few requirements since it's still a beta block:
* For Jetpack sites, add `define( 'JETPACK_BETA_BLOCKS', true );` to your `wp-config.php`
* For WP.com sites, either be proxied or add a sticker to the site (ask me)
* For Atomic sites, it's complicated - no need to test that case here

It's worth noting that enabling beta blocks works retroactively - posts already uploaded will now render correctly, no content is lost.

Also worth noting that just now there is a known issue preventing playing a story on WP.com, currently being worked on.

PR submission checklist:
